Aptoide/aptoide-client-v8
# [DT-21] implement install manager for DT

**What does this PR do?**

   Adds InstallManager implementation for DT service layer.

**Database changed?**

   Yes

**Where should the reviewer start?**

- Better by commits

**How should this be manually tested?**

  This is not connected to UI, so essentially just run the app and make sure no new errors in logs

**What are the relevant tickets?**

  Tickets related to this pull-request: [DT-21](https://aptoide.atlassian.net/browse/DT-21)



**Code Review Checklist**
